src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final int FIRST_SALARY_SPLIT_INDEX = 2;
+    private static final int SECOND_SALARY_SPLIT_INDEX = 3;
+    private static final int NAME_SPLIT_INDEX = 1;
+    private static final int DATE_SPLIT_INDEX = 0;
+    private static final String WHITESPACE = " ";
+    private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder();
+        builder.append("Report for period " + dateFrom + " - " + dateTo);
+        int[] salary = new int[names.length];
+        try {
+            Date firstDate = SIMPLE_DATE_FORMAT.parse(dateFrom);
```

Comment:
[Use LocalDate instead of Date.](https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#use-localdate-instead-of-date)